ml-agents/mlagents/trainers/ppo/trainer.py

```diff
@@ -70,7 +70,7 @@ def __init__(
         self.load = load
         self.multi_gpu = multi_gpu
         self.seed = seed
-        self.policy: TFPolicy = None
+        self.policy: PPOPolicy = None  # type: ignore
```

Comment:
Ignore the error of setting this to None, since we'll always set it to an instance before using.